cleanlab/multiannotator.py

```diff
@@ -275,26 +300,439 @@ def get_label_quality_multiannotator(
+        Boolean to specify if `detailed_label_quality` is returned.
+    return_annotator_stats : bool, default = True
+        Boolean to specify if `annotator_stats` is returned.
+    return_weights : bool, default = False
+        Boolean to specify if `model_weight` and `annotator_weight` is returned.
+    verbose : bool, default = True
+        Important warnings and other printed statements may be suppressed if ``verbose`` is set to ``False``.
+    label_quality_score_kwargs : dict, optional
+        Keyword arguments to pass into :py:func:`get_label_quality_scores <cleanlab.rank.get_label_quality_scores>`.
+
+    Returns
+    -------
+    labels_info : dict
+        Dictionary containing up to 3 pandas DataFrame with keys as below:
+
+        ``label_quality`` : pandas.DataFrame
+            pandas DataFrame in which each row corresponds to one example, with columns:
+
+            * ``num_annotations``: the number of annotators that have labeled each example.
+            * ``consensus_label``: the single label that is best for each example.
+            * ``annotator_agreement``: the fraction of annotators that agree with the consensus label (only consider the annotators that labeled that particular example).
+            * ``consensus_quality_score``: label quality score for consensus label, calculated using the crowdlab method.
+
+        ``detailed_label_quality`` : pandas.DataFrame (returned if `return_detailed_quality=True`)
+            Returns a pandas DataFrame with columns `quality_annotator_1`, `quality_annotator_2`, ..., `quality_annotator_M` where each entry is
+            the label quality score for the labels provided by each annotator (is ``NaN`` for examples which this annotator did not label).
+
+        ``annotator_stats`` : pandas.DataFrame (returned if `return_annotator_stats=True`)
+            Returns overall statistics about each annotator, sorted by lowest annotator_quality first.
+            pandas DataFrame in which each row corresponds to one annotator (the row IDs correspond to annotator IDs), with columns:
+
+            * ``annotator_quality``: overall quality of a given annotator's labels, calculated using the crowdlab method`.
+            * ``num_examples_labeled``: number of examples annotated by a given annotator.
+            * ``agreement_with_consensus``: fraction of examples where a given annotator agrees with the consensus label.
+            * ``worst_class``: the class that is most frequently mislabeled by a given annotator.
+
+        ``model_weight`` : np.ndarray (returned if `return_weights=True`)
+            An array of shape ``(P,)`` where is the number of models in the ensemble, specifying the model weight used in weighted averages
+
+        ``annotator_weight`` : np.ndarray (returned if `return_weights=True`)
+            An array of shape ``(M,)`` where M is the number of annotators, specifying the annotator weights used in weighted averages
```

Comment:
may be better to just point user to refer to the documentation of `get_label_quality_multiannotator`, except for the args which now have different meanings in this function.

Reply:
Not sure what is the cleanest way to reference the docstring here. Currently I am listing every arg individually and pasting the same reference on each line. It is a lot of repeated references but my idea was to still list every argname (thought it'll be nice to list them one by one). 
But if it is too much repetition I could remove the individual args and just reference the entire function as a whole.

Reply:
Yes let's list every arg individually, specify its type, and give one phrase  description about what it is. This phrase should immediately be followed by:

> ..., in the same format as expected by the [get_label_quality_multiannotator](link) function.
 
A good example of this is the docstrings for individual label quality scores, cf:

https://docs.cleanlab.ai/stable/cleanlab/rank.html#cleanlab.rank.get_confidence_weighted_entropy_for_each_label

https://docs.cleanlab.ai/stable/cleanlab/rank.html#cleanlab.rank.get_normalized_margin_for_each_label